libhb/work.c

```diff
@@ -1521,15 +1403,18 @@ static void do_job(hb_job_t *job)
         goto cleanup;
     }
 
-    // sanitize_qsv looks for subtitle render filter, so must happen after
-    // sanitize_subtitle
-    result = sanitize_qsv(job);
-    if (result)
+#if HB_PROJECT_FEATURE_QSV
+    if (!job->qsv.ctx)
     {
-        *job->done_error = HB_ERROR_WRONG_INPUT;
-        *job->die = 1;
-        goto cleanup;
+        job->qsv.ctx = av_mallocz(sizeof(hb_qsv_context));
```

Comment:
Just a comment wondering how others feel about this.  job->qsv.ctx is allocated unconditionally whether QSV is used or not.  Not a big deal IMO, but could be a bit misleading to someone new reading the code.

Reply:
Good point, thanks! Fixed.